pageserver/src/tenant/remote_timeline_client/manifest.rs

```diff
@@ -0,0 +1,45 @@
+use chrono::NaiveDateTime;
+use serde::{Deserialize, Serialize};
+use utils::{id::TimelineId, lsn::Lsn};
+
+#[derive(Clone, Serialize, Deserialize)]
+pub struct TenantManifest {
+    /// Debugging aid describing the version of this manifest.
+    pub version: usize,
```

Comment:
Maybe it will be useful for backwards compatibility as well, but with Serde, I think it's easiest to evolve the schema by adding optional fields (decoding to `None` if the field is not present), and then handling it manually if the field is `None` (say in the `from_manifest` function).

Reply:
Yep, that's generally preferable, but it's sometimes necessary to make backwards-incompatible changes, and this `version` field would help with that. Anyway, consider just mentioning that it should be bumped as needed with backwards-incompatible changes.